tests/acceptance/features/webUIFiles/copyPrivateLinks.feature

```diff
@@ -10,10 +9,9 @@ Feature: copy current path as a permanent link
     And the user has browsed to the files page
 
   Scenario Outline: Copy permalink to clipboard
-    When the user browses to the folder <folder_name> on the files page
-    And the user copies the permalink of the current folder using the webUI
-    Then the clipboard content should match permalink of resource <folder_name>
+    When the user copies the permalink of the <resource> <resource_name> using the webUI
```

Comment:
```suggestion
    When the user copies the permalink of the resource <resource_name> using the webUI
```

that function takes only one argument, no need to make the sentence look more complicated